app/src/main/contrib/components/ChooseGroup/ChooseGroupContent.tsx

```diff
@@ -6,19 +6,22 @@ import { GroupList } from "./GroupList";
 import { ButtonLink } from "../../../shared/components/ButtonLink";
 import { modellingGroupsActions } from "../../actions/modellingGroupsActions";
 import { LoadingElement } from "../../../shared/partials/LoadingElement/LoadingElement";
-// import { GlobalState } from "../../reducers/reducers";
+
+import { Dispatch } from "redux";
 
 import "../../../shared/styles/common.scss";
+import {ContribAppState} from "../../reducers/contribReducers";
+import {doNothing} from "../../../shared/Helpers";
 
 export interface ChooseGroupProps {
     groups: ModellingGroup[];
-    dispatch: any;
+    getGroups: () => Promise<void>;
```

Comment:
Action get groups is not returning promise.

Reply:
Typescript thinks it is? Because it's async?